chartfx-acc/src/main/resources/public/main.css

```diff
@@ -0,0 +1,315 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Tahoma, Arial, sans-serif;
+  position: relative;
+  min-height: 100%;
+}
+
+body {
+  margin: 0;
+  font-family: Tahoma, Arial, sans-serif;
+  padding: 0 0 40px 0;
+  color: #333;
+  background: #f9f9f9;
+}
+
+h1, h2, h3, h4 {
+  font-family: monospace;
+  font-weight: 300;
+  color: #444;
+}
+
+small {
+  color: #555;
+}
+
+header {
+  background: #00599c;
+  border-bottom: 5px solid #ffb446;
+  box-shadow: 0 1px 0 0 rgba(0, 0, 0, .6);
+}
+
+nav {
+  padding: 15px;
+  margin: 0 auto;
+  max-width: 1200px;
+  position: relative;
+}
+
+nav #menu {
+  font-family: Comic Sans MS, Tahoma, Arial, sans-serif;
+  margin-top: 20px;
+  float: right;
+}
+
+a {
+  text-decoration: none;
+  color: #008ab4;
+}
+
+#menu li {
+  float: left;
+  margin: 0 10px;
+}
+
+#menu li a, #logout {
+  background: transparent;
+  cursor: pointer;
+  border: 0;
+  font-size: 16px;
+  display: inline-block;
+  color: #fff;
+  font-family: Comic Sans MS, Tahoma, Arial, sans-serif;
+  text-align: center;
+  height: 30px;
+  line-height: 30px;
+  padding: 0 10px;
+  text-decoration: none;
+}
+
+#logo {
+  max-height: 50px;
+}
+
+#chooseLanguage {
+  top: 15px;
+  right: 35px;
+  position: absolute;
+}
+
+#chooseLanguage li {
+  float: left;
+}
+
+#chooseLanguage button {
+  cursor: pointer;
+  margin-left: 8px;
+  width: 18px;
+  height: 18px;
+  border-radius: 9px;
+  opacity: 0.6;
+  border: 1px solid #222;
+  background-size: 100%;
+}
+
+#chooseLanguage button:hover {
+  opacity: 0.8;
+}
+
+#main {
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 25px;
+}
+
+#content {
+  padding: 15px 20px;
+  background: #fff;
+  box-shadow: 0 1px 5px rgba(0, 0, 0, 0.25);
+}
+
+#footer {
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  height: 50px;
+  line-height: 18px;
+  width: 100%;
+  text-align: center;
+  background: #fff;
+  border-top: 2px solid #ffb446;
+  font-size: 14px;
+}
+
+nav ul, nav li {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+/* Needlessly fancy menu hover effect */
+#menu li {
+  position: relative;
+}
+
+#menu li a::after, #logout:after {
+  position: absolute;
+  top: 28px;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  background: rgba(255, 255, 255, 0.5);
+  border-radius: 5px;
+  content: '';
+  opacity: 0;
+  -webkit-transition: opacity 0.3s, -webkit-transform 0.3s;
+  transition: opacity 0.3s, transform 0.3s;
+  -webkit-transform: translateY(10px);
+  transform: translateY(10px);
+}
+
+#logout:hover::after, #logout:focus::after, #menu li a:hover::after, #menu li a:focus::after {
+  opacity: 1;
+  -webkit-transform: translateY(0px);
+  transform: translateY(0px);
+}
+
+/* Very basic grid */
+.row {
+  width: 100%;
+  overflow: auto;
+}
+
+.row>* {
+  float: left;
+}
+
+.row-2 .col {
+  width: 49%;
+}
+
+.row-2 .col:nth-child(odd) {
+  margin: 0% 1% 0% 0%;
+}
+
+.row-2 .col:nth-child(even) {
+  margin: 0% 0% 0% 1%;
+}
+
+.row-3 .col {
```

Comment:
Codacy found an issue: [Expected selector ".row-3 .col" to come before selector ".row-2 .col:nth-child(even)" (no-descending-specificity)](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=6305992)